doc/documentation.md

```diff
@@ -227,6 +230,10 @@ messages for the session manager
     - `Error error` - the error that occurred
     - `Object details` - the details about the error
 
+`sdk.on('disconnected', () => {})` - the underlying websocket connection has
```

Comment:
shouldn't that be `sdk.on('disconnect'` ?

Reply:
no, the event coming from realtime is `disconnect`, the one that gets forwarded is `disconnected`.

Reply:
Yep, I mixed it up

Reply:
Looks like `disconnected` is still being emitted.

Reply:
No, we're still going to emit `disconnected`, which is changed from the underlying websocket lib which emits `disconnect` instead.